pathfinder/tests/fixtures/core.py

```diff
@@ -1,73 +1,73 @@
 import random
-from typing import List, Dict, Tuple
+from typing import List
 from unittest.mock import Mock
 
 import pytest
-from _pytest import monkeypatch
 from eth_utils import to_checksum_address, encode_hex, keccak
 from raiden_libs.contracts import ContractManager
 from web3 import Web3
+from web3.contract import Contract
 
-from pathfinder.contract.token_network_contract import TokenNetworkContract
 from pathfinder.pathfinding_service import PathfindingService
 from pathfinder.token_network import TokenNetwork
-from pathfinder.utils.types import ChannelId, Address
 
 
 @pytest.fixture
 def populate_token_networks(
     token_networks: List[TokenNetwork],
-    token_network_contracts: List[TokenNetworkContract],
-    monkeypatch: monkeypatch
+    token_network_contracts: List[Contract],
 ) -> None:
 
-    def get_channel_deposits_patched(
-        self: TokenNetworkContract,
-        channel_id: ChannelId
-    )-> Dict[Address, int]:
+    for token_network in token_networks:
+        # some random magic to add channels
+        for channel_id in range(10):
+            random.seed(channel_id)
```

Comment:
You should not initialize the random seed more than once in a program. Just initialize it once somewhere in the program and either use the system time or leave it blank for usage os `os.urandom()`. Docs [here](https://docs.python.org/3.6/library/random.html#random.seed)